src/Tribe/Main.php

```diff
@@ -3940,21 +3034,6 @@ public function eventMetaBox() {
 			include( $this->plugin_path . 'src/admin-views/event-sidebar-options.php' );
 		}
 
-		/**
-		 * Get the date string (shortened).
-		 *
-		 * @param string $date The date.
-		 *
-		 * @return string The pretty (and shortened) date.
-		 */
-		public function getDateStringShortened( $date ) {
```

Comment:
Brought back and deprecated. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3259))